scripts/uberenv/spack_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -10,123 +10,140 @@ packages:
       mpi: [mvapich2]
       gl: [opengl]
       glu: [openglu]
-
   opengl:
-    paths:
-      opengl@1.7.0: /usr
-    buildable: False
+    buildable: false
```

Comment:
This is just an update to the syntax of this file. @becker33 gave me a script to do this so I'll run the rest through it.  Thanks Greg!

Reply:
I want this script!

Reply:
I emailed this earlier to a few people, @adrienbernede included..